openwisp_notifications/static/openwisp-notifications/js/notifications.js

```diff
@@ -350,7 +352,7 @@ function initWebSockets($) {
             toast.slideDown('slow', function () {
                 setTimeout(function () {
                     toast.slideUp('slow', function () {
-                        toast.remove();
+                        // toast.remove();
```

Comment:
What is this remove?

Reply:
It was not ready that why I was not moving to it ``ready`` column :smile:  And I commented this to restyle toast message. I will uncomment it when it gets ready.

>Do we want to merge this to master or create a branch? @nemesisdesign

I had created one branch named ``new-menu`` like we were making in other modules.